### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by tduffy83**
Added instructions for installing prerequisites.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @tduffy83, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;supervised-installer#114).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: README.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ This installation method provides the full Home Assistant experience on a regula
 By not using the Home Assistant Operating System, the user is responsible for making sure that all required components are installed and maintained. Required components and their versions will change over time. Home Assistant Supervised is provided as-is as a foundation for community supported do-it-yourself solutions. We only accept bug reports for issues that have been reproduced on a freshly installed, fully updated Debian with no additional packages.
 
 This method is considered advanced and should only be used if one is an expert in managing a Linux operating system, Docker and networking.
+## Prerequesite Installation
+
+Run as root (sudo su):
+```bash
+apt install jq avahi-daemon network-manager
```
Comment (frenck) [inline README.md:16]: This list is incomplete, it is listed in the link on line 3 already, including all other requirements. So I'm not sure if we should include it like this.

This installation method is aimed at expert users, would they really need this with the requirements listed in the link above?

**Comment by tduffy83** [comment]
How is it incomplete?  There are three packages installed and then I direct the user to the docker ce repository install instructions which includes everything else you need.  Honestly I'm an expert user but had to take way too much time figuring this out.  I don't see how having a clear install instruction which includes the prerequisites hurts anything.  It seems like to me you are trying to steer people towards hassos by making this supervised method unnecessarily hard to follow.  The fact is that hassos is not there yet and there are many use cases that require a full fat OS.

**Comment by frenck** [comment]
>  Honestly I'm an expert user but had to take way too much time figuring this out. 

It is directly listed when clicking the first link on the repo readme?

image

The point is, is that the current instructions you are adding might assume that that is all is needed, which is not true and therefore incomplete. For example, depending on your how the OS was installed, you might need to install some additions, for example, dbus, curl and apparmor-utils, but also additional configuration of Docker is needed.

IMHO, this should be either complete or we should not do it at all. I strongly believe the requirements linked to should be enough for a user to get this installed. If that isn't enough, this installation method is probably not for you.

**Comment by tduffy83** [comment]
I installed Debian from CD image and unchecked everything except for ash server so it's as bare as can be.  Curl is installed when you install docker, apparmor is in the kernel, and dbus is installed with Debian.  In terms of docker config I admit I missed that, but the install script worked out of the box and the supervisor does not give any warnings as not being supported.

**Comment by tduffy83** [comment]
https://github.com/home-assistant/operating-system/issues/872#issuecomment-695829492

This is why I needed to install supervised in the first place.  If hassos worked for me I wouldn't even have suggested this edit.  I understand that the devs don't like supporting the supervised install method but as long as there are use cases that make it necessary my views are that it should be made as painless to install as possible.
